client/src/Movies/SavedList.js

```diff
@@ -1,18 +1,15 @@
 import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
 
 export default class SavedList extends Component {
-  constructor(props) {
-    super(props);
-  }
-
   render() {
     return (
       <div className="saved-list">
         <h3>Saved Movies:</h3>
         {this.props.list.map(movie => (
           <span className="saved-movie">{movie.title}</span>
         ))}
-        <div className="home-button">Home</div>
+        <Link to='/'className="home-button">Home</Link>
       </div>
     );
   }
```

Comment:
You should go back and make a checker for duplicates. Keep challenging yourself Jonathan.